### ISSUE TITLE
'any' is not a valid type

### ISSUE THREAD

**Issue by broofa** [chaliy/sequelize-json-schema#23]
JSON and JSONB attributes currently map to JSON-Schema properties of type `"any"`, which is not a valid type.  This causes AJV to throw errors like this:
> data.properties['some_property'].type should be equal to one of the allowed values

JSON types should map to `{type: ['array', 'object', 'string', 'number', 'boolean', 'null']}`

**Comment by RickEyre** [comment]
@broofa I see you've been contributing to your own fork. There's no activity on this repository anymore and there are a few features I want to add too. Support for `defaultValue` for example and validations. 

Would you be interested in publishing your fork?

**Comment by bravo-kernel** [comment]
How about joining forces here, I have the same items on my todo list for https://github.com/alt3/sequelize-to-json-schemas.

**Comment by broofa** [comment]
FYI, I'm heading out on vacation so don't have time to reply at the moment.  I'll be back after next week.

(For what it's worth, I'd prefer to get this module to a place where it's well-maintained rather than forking.  We just need @chaliy to cooperate which, hopefully, he's willing to do.  I've reached out to him about adding others as owners/maintainers, and cc'ed node support on that.  If he doesn't engage in the conversation(s) we're having about this, I can move forward with NPM to see about migrating ownership, but that should be a last-resort option.)

Feel free to continue the conversation. I'll catch up when I get back.

**Comment by RickEyre** [comment]
@broofa @bravo-kernel I agree on getting this fork working as the main option. It seems like @chaliy has lost interest in maintaining this package so moving forward with NPM is probably what will have to happen.

**Comment by bravo-kernel** [comment]
@RickEyre no problem, all the best reviving this repo. I will push forward on the alt3 repo myself as it basically is a drop-in replacement and offers a lot of added benefits for future maintenance. 👍

**Comment by bravo-kernel** [comment]
If you guys change your mind at any time let me know and I will add you as repo maintainers.

**Comment by RickEyre** [comment]
@bravo-kernel I'm open to contributing to your repository as well. The fact is that my team needs this functionality now, not later. If I can accomplish that through your repository I'm cool with that too.

**Comment by bravo-kernel** [comment]
@RickEyre as you can probably see [here](https://github.com/alt3/sequelize-to-json-schemas/blob/master/lib/type-mapper.js#L344) these kind of additions become quite simple once a solid core is in place. Writing (and updating) tests has become simple as well now that json strings are no longer needed as can be seen [here](https://github.com/alt3/sequelize-to-json-schemas/blob/master/test/strategy-interface.test.js#L147).  Released as 0.1.3.

Perhaps you can take this as inspiration for validations and go for it 💃

**Comment by chaliy** [comment]
@broofa now has all permissions needed to publish to npm. If you guys reach some agreement on ownership, I can transwer ownership to selected person.
